README.md

```diff
@@ -180,6 +174,19 @@ $ grunt bump --setversion=2.0.1
 >> Pushed to origin
 ```
 
+If you want to append commit message content, you can use the ```commitmsg``` tag in the command line.
```

Comment:
the commit msg should be about the release. the commit messages on the commits leading up to the release should be about whats in it. Those ideally should get put into some kind of changelog.